src/github.com/getlantern/checkfallbacks/checkfallbacks.go

```diff
@@ -149,6 +150,7 @@ func testFallbackServer(fb *client.ChainedServerInfo, workerId int) (output full
 		output.info = []string{"\n" + string(reqStr)}
 	}
 
+	req.Header.Set("X-LANTERN-AUTH-TOKEN", fb.AuthToken)
```

Comment:
Interesting. During my test, `checkfallbacks` never succeeded without this line.